cmd/start.go

```diff
@@ -57,10 +58,12 @@ func init() {
 		syncProviderFlagName, "y", "", "DEPRECATED: Set a sync provider e.g. filepath or remote",
 	)
 	flags.StringP(logFormatFlagName, "z", "console", "Set the logging format, e.g. console or json ")
+	flags.StringP(grpcCertPath, "g", "", "Path to TLS certificate to be used by grpc sync provider")
```

Comment:
Can you document here or elsewhere that if not provided, root/system certs will be used? Maybe use the same language as the underlying go libraries.